platform-ajax/src/test/java/com/softicar/platform/ajax/dom/event/wheel/AjaxDomWheelEventTest.java

```diff
@@ -0,0 +1,155 @@
+package com.softicar.platform.ajax.dom.event.wheel;
+
+import com.softicar.platform.ajax.dom.event.AbstractAjaxDomEventTestDiv;
+import com.softicar.platform.ajax.testing.selenium.engine.level.low.AbstractAjaxSeleniumLowLevelTest;
+import com.softicar.platform.common.core.thread.sleep.Sleep;
+import com.softicar.platform.dom.event.DomEventType;
+import com.softicar.platform.dom.event.IDomEvent;
+import com.softicar.platform.dom.event.IDomWheelEventHandler;
+import java.time.Duration;
+import java.util.Collection;
+import java.util.Iterator;
+import org.junit.Test;
+
+public class AjaxDomWheelEventTest extends AbstractAjaxSeleniumLowLevelTest {
+
+	@Test
+	public void testMouseWheelDown() {
+
+		var testDiv = openTestNode(() -> new TestDiv());
+
+		mouseWheelDown(testDiv);
+		waitForServer();
+
+		var events = testDiv.getEvents();
+		new WheelEventAsserter(events)//
+			.nextEvent()
+			.assertDeltaXZero()
+			.assertDeltaYPositive()
+			.assertDeltaZZero()
```

Comment:
shouldn't we simplify this to `assertWheelDown`?

Reply:
yep